sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -1214,9 +1213,9 @@ def __init__(
 
   def setup(self):
     # type: () -> None
+    _LOGGER.debug('Setup called for %s', self)
```

Comment:
We can probably get rid of this line for consistency.

Reply:
Done, thanks